src/routes/_index/component/eek/+page.marko

```diff
@@ -360,9 +360,6 @@
         </svg>
     </div>
     </highlight-code>
-
```

Comment:
Rogue diff

Reply:
Actually its intentional, there were 3 new lines at the end of the file. Looked bad. Part of it came about that there were new docs, but I think this is cleaner